src/hexen/sb_bar.c

```diff
@@ -1180,9 +1233,11 @@ void DrawMainBar(void)
          for (j = 0; j <= crispy->hires; j++)
           for (k = 0; k <= crispy->hires; k++)
           {
-            I_VideoBuffer[((i << crispy->hires) + j) * SCREENWIDTH + ((95 << crispy->hires) + k)] = 0;
-            I_VideoBuffer[((i << crispy->hires) + j) * SCREENWIDTH + ((96 << crispy->hires) + k)] = 0;
-            I_VideoBuffer[((i << crispy->hires) + j) * SCREENWIDTH + ((97 << crispy->hires) + k)] = 0;
+            temp = SCREENWIDTH * ((i << crispy->hires) + j);
```

Comment:
This change it not really widescreen-related, right? Please add comment.

Reply:
I went ahead and removed the use of temp.